content/docs/upgrading/upgrade.md

```diff
@@ -4,11 +4,120 @@ description = "Upgrading your deployment to a later version of Kubeflow"
 weight = 30
 +++
 
-Until version 1.0, Kubeflow makes no promises of backwards compatibility or 
-upgradeability. There is **no clean upgrade path** to the latest version of 
-Kubeflow ({{% kf-latest-version %}}).
+## Upgrading from Kubeflow v0.7.0 or later
```

Comment:
How about adding some sort of banner that upgrades are currently in "alpha" with limited support?